### ISSUE TITLE
Cellar: section about objects versionning 

### ISSUE THREAD

**Issue by BlackYoup** [CleverCloud/documentation#55]
It can be used as a backup solution but it needs:
- to be set up before using the bucket
- to be handled by the code to avoid storing too many versions of current / deleted objects

**Comment by BlackYoup** [comment]
- What is bucket versioning
- Its applications / why use it
- How to configure it
- How to use it

Something like that?

**Comment by Dekabry** [comment]
Could you provide a very raw draft ? I'm down to write it up.
We can also have a discord chat if you'd prefer so i get most of the info

**Comment by Kirbeerus** [comment]
I managed to make versioning work on cellar with the AWS CLI.
I am planning to make add it to the doc.
About this `to be handled by the code to avoid storing too many versions of current / deleted objects`
Do you mean that it's the user that need to handle in their code the versioning so that they aren't too many version of one file ?

**Comment by BlackYoup** [comment]
> Do you mean that it's the user that need to handle in their code the versioning so that they aren't too many version of one file ?

Yes, storing multiple versions adds up in term of storage space so it's something to keep in mind.
